src/mockup-entry.ts

```diff
@@ -19,3 +20,11 @@ class MockupWikibaseContentLanguages {
 ( window as MwWindow ).wb = {
 	WikibaseContentLanguages: MockupWikibaseContentLanguages,
 };
+
+( window as MwWindow ).$ = {
+	uls: {
+		data: {
+			getDir: ( code: string ) => directionalities.default[ code ],
```

Comment:
I would have written a fake implementation that would not have required the additional json file, but I think this is ok too.